### ISSUE TITLE
TWRP does not become permanent without installing magisk on xiaomi mi a1

### ISSUE THREAD

**Issue by Francygal2001** [CosmicDan-Android/android_device_xiaomi_tissot#4]
hi there is this problem that, when i flash the file that is needed to have permanent TWRP flashing using temporary TWRP, it does not become permanent without then installed magisk, what is the problem?

**Comment by cosmicdan** [comment]
Not sure sorry. Is this on Pie? There are all kinds of small problems on Pie that I cannot fix (yet). I will hopefully figure it out one day.

**Comment by Francygal2001** [comment]
yes I have android pie, in fact I can't install an sGSI, if there is another method to install I would be very grateful

**Comment by cosmicdan** [comment]
I don't know, I've not been active on Tissot development for some time - you'd best be browsing/asking on XDA.
